CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 2.4.2
+
+### New features
+
+* An option to toggle the manager window with the tray icon (the 2.3 behavior useful when there are no window decorations for closing it)
```

Comment:
Looks like I got confused there as this targeted main. Shall we pick it into 2.4.2, @infirit? :shrug: